app/components/primer/auto_complete.rb

```diff
@@ -83,6 +129,8 @@ class AutoComplete < Primer::Component
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
     def initialize(src:, id:, **system_arguments)
       @id = id
+      @aria_label = system_arguments[:"aria-label"] || system_arguments.dig(:aria, :label)
+      system_arguments.delete(:"aria-label") && system_arguments[:aria]&.delete(:label)
```

Comment:
`<auto complete aria-label="blah">` will fail [aria-allowed-attr][0] so we need to delete it. 

Note: `<auto-complete>` is a custom web component element so I'm not entirely sure whether this is a false positive. However, does not seem *critical* to have this label so going to keep this change rather than disabling the rule entirely. 

[0]: https://dequeuniversity.com/rules/axe/4.1/aria-allowed-attr